tests/Database/DatabaseEloquentIntegrationTest.php

```diff
@@ -377,7 +377,7 @@ public function testFindOrFailWithMultipleIdsThrowsModelNotFoundException()
     {
         EloquentTestUser::create(['id' => 1, 'email' => 'taylorotwell@gmail.com']);
         EloquentTestUser::findOrFail([1, 2]);
-    }
+    }    
```

Comment:
Thanks! I'd been looking for a test of the save method and passing through $options in particular and must have tweaked the file inadvertantly!